pkg/k8sutil/k8sutil.go

```diff
@@ -179,6 +179,6 @@ func isKubeMetaKind(kind string) bool {
 	return false
 }
 
-func isRunModeLocal() bool {
```

Comment:
This does not need to be exported. You can just do
```go
func addMetrics(ctx context.Context, cfg *rest.Config, namespace string) {
	if err := serveCRMetrics(cfg); err != nil {
		if errors.Is(err, k8sutil.ErrRunLocal) {
			log.Info("Skipping CR metrics server creation; not running in a cluster.")
			return
		}
		...
	}
	...
}
```

Reply:
you are right.